org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CallHierarchyHandler.java

```diff
@@ -221,13 +249,20 @@ private IMember getCallHierarchyElement(String uri, int line, int character, IPr
 
 		List<CallHierarchyOutgoingCall> result = new ArrayList<>();
 		for (MethodWrapper call : calls) {
+			Collection<CallLocation> callLocations = call.getMethodCall().getCallLocations();
+			List<Range> ranges = toCallRanges(callLocations);
```

Comment:
again, `List<Range> ranges = toCallRanges(callLocations);` needs to go inside the if block.

Reply:
Could you, please, check it again?